policy.go

```diff
@@ -40,7 +40,8 @@ type Policy struct {
 }
 
 type Rotation struct {
-	Interval int `json:"interval_month,omitempty"`
+	Enabled  *bool `json:"enabled,omitempty"`
```

Comment:
To handle Disabling of rotation policy better, we discussed about a new struct RotationPolicy with both attributes as pointer values. Work with @EktaBharti1998 on this.